app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ErrorsController < ActionController::Base
+  include PrxAuth::Rails::Controller
```

Comment:
If the user is authenticated and gets a 404, we'll get the `user_id` in the log.info statement.